modules/xmpp/strophe.jingle.js

```diff
@@ -307,8 +306,7 @@ class JingleConnectionPlugin extends ConnectionPlugin {
         this.connection.sendIQ(
             $iq({ type: 'get',
                 to: this.connection.domain })
-                .c('services', { xmlns: 'urn:xmpp:extdisco:1' })
-                .c('service', { host: `turn.${this.connection.domain}` }),
```

Comment:
You are removing the turn service?

Reply:
No, not at all. This change modifies the request to an XMPP server to retrieve a list of service addresses. The response will include all service addresses that are configured at the server. This code seems to add a specific server to that request, which doesn't make much sense. This could be intended as a kind of filter ("give me the details of server `turn.example.org`"), but that's not how XEP-0215, nor the Prosody implementation of it, works.